nilearn/plotting/displays.py

```diff
@@ -1216,6 +1221,9 @@ def add_graph(self, adjacency_matrix, node_coords,
                               vmin=edge_vmin, vmax=edge_vmax,
                               **edge_kwargs)
 
+        if colorbar:
+            self._colorbar_show(ax, threshold=edge_threshold)
```

Comment:
> Given the code I have and given the number of different classes inheritances I have. I have no other choice to move forward.

I don't see anything nasty here... He is using a method of its own class, this is totally legit.

The question is whether we accept to have this hack until somebody decide to clean this code.